biztalk/core/sftp-adapter.md

```diff
@@ -183,7 +188,7 @@ When finished, your configuration looks similar to the following:
 |Does the SFTP adapter support 256-bit encryption?|Yes - The SFTP adapter supports 256-bit encryption. The supported encryption algorithms include:<br /><br /> - AES encryption: 256-bit, 192-bit, or 128-bit SDCTR or CBC<br /><br /> - 3DES (Triple-DES) encryption: 168-bit SDCTR or CBC|
 |What SSH versions does the adapter support?|Only SSH2. Connection cannot be established with SFTP servers having SSH1 version.|
 |Is file mask case sensitive?|No. *.txt and \*.TXT works alike. Please install the latest cumulative update for BizTalk Server 2013. BizTalk Server 2013 RTM release had case-sensitive file masks.|
-|Exporting bindings give a blank password field. When trying to create a receive location by importing these bindings what all changes are to be made?|Edit the binding file by editing the password field. Also, in `<Password vt="1">MySecretPassword</Password>`, **vt=”1”** indicates a null value. Change that to **vt=”8”**, which indicates a string. For example:<br /><br /> `<Password vt="8">MySecretPassword</Password>`<br /><br /> For more details, see [https://msdn.microsoft.com/library/system.runtime.interopservices.varenum(v=vs.100).aspx](https://msdn.microsoft.com/library/system.runtime.interopservices.varenum\(v=vs.100\).aspx)|
+|Exporting bindings give a blank password field. When trying to create a receive location by importing these bindings what all changes are to be made?|Edit the binding file by editing the password field. Also, in `<Password vt="1">MySecretPassword</Password>`, **vt=”1”** indicates a null value. Change that to **vt=”8”**, which indicates a string. For example:<br /><br /> `<Password vt="8">MySecretPassword</Password>`<br /><br /> For more details, see [https://learn.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.varenum](https://learn.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.varenum)|
```

Comment:
Link to docs must be a relative link, not an absolute link with the "en-us" locale:

```suggestion
|Exporting bindings give a blank password field. When trying to create a receive location by importing these bindings what are all the changes to be made? | Edit the binding file by editing the password field. Also, in `<Password vt="1">MySecretPassword</Password>`, **vt=”1”** indicates a null value. Change that to **vt=”8”**, which indicates a string. For example:<br /><br /> `<Password vt="8">MySecretPassword</Password>`<br /><br /> For more details, see [VarEnum Enum](/dotnet/api/system.runtime.interopservices.varenum). |
```